design-tokens/tokens/layer1/colors.json

```diff
@@ -1,392 +1,404 @@
 {
   "color": {
-    "black": {
-      "value": "#000000",
-      "type": "color"
-    },
-    "white": {
-      "value": "#ffffff",
-      "type": "color"
-    },
-    "gray": {
-      "100": {
-        "value": "#f7fafc",
-        "type": "color"
-      },
-      "200": {
-        "value": "#edf2f7",
-        "type": "color"
-      },
-      "300": {
-        "value": "#e2e8f0",
-        "type": "color"
-      },
-      "400": {
-        "value": "#cbd5e0",
-        "type": "color"
-      },
-      "500": {
-        "value": "#a0aec0",
+    "support": {
```

Comment:
Should we add transparent to support colors?

Reply:
We can. This would be something like:
**Layer 1**
support - transparent
**Layer 2**
bg--none  - transparent

Right?

Reply:
yeah looks good